rust_src/remacs-sys/lib.rs

```diff
@@ -124,6 +124,17 @@ pub enum PseudovecType {
     PVEC_FONT, /* Should be last because it's used for range checking.  */
 }
 
+#[repr(isize)]
```

Comment:
Based on my current understanding of Rust FFI, I think we should use `repr(C)` for all our enums.

> It has fairly simple intent: do what C does. The order, size, and alignment of fields is exactly what you would expect from C or C++. Any type you expect to pass through an FFI boundary should have repr(C), as C is the lingua-franca of the programming world. 
>
> -- https://doc.rust-lang.org/nomicon/other-reprs.html

The C enum in question is:

```c
enum text_cursor_kinds
{
  DEFAULT_CURSOR = -2,
  NO_CURSOR = -1,
  FILLED_BOX_CURSOR,
  HOLLOW_BOX_CURSOR,
  BAR_CURSOR,
  HBAR_CURSOR
};
```

What size does this have?

> An enum is only guaranteed to be large enough to hold int values. The compiler is free to choose the actual type used based on the enumeration constants defined so it can choose a smaller type if it can represent the values you define.
> 
> -- https://stackoverflow.com/a/366026/509706

So I think `repr(C)` ensures that our enums match the size the C codebase expects.